boot/bootstate20.go

```diff
@@ -211,7 +211,8 @@ func (u20 *bootStateUpdate20) commit(markedSuccessful bool) error {
 		}
 	}
 
-	resealOpts := ResealKeyToModeenvOptions{}
+	// MarkSuccessful does not do the switch TryModel -> Model
```

Comment:
this comment is a bit confusing because I suppose commit is called by more than by MarkSuccessful

Reply:
Yes. But all the other cases also do not update the model. I will rephrase it.